sorc/build_all.sh

```diff
@@ -82,20 +64,94 @@ while getopts ":a:dfghj:kuvwy" option; do
       ;;
   esac
 done
-
 shift $((OPTIND-1))
 
+# If no build system was specified, build for gfs forecast-only
+if [[ $# -eq 0 ]]; then
+   selected_systems="gfs"
+else
+   selected_systems="$*"
+fi
+
+supported_systems=("gfs" "gefs" "sfs" "gsi" "gdas" "all")
+# shellcheck disable=SC2034
+gfs_builds="gfs gfs_utils ufs_utils upp ww3_gfs"
```

Comment:
Would call the UFS build `ufs_gfs`, `ufs_gefs`, etc. to avoid confusion.

Reply:
Could also use an associative array to hold the builds. Looks cleaner than a bunch of variable names:
```suggestion
builds['gfs']="ufs_gfs gfs_utils ufs_utils upp ww3_gfs"
```

Reply:
Done.